extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/dialect/QuarkusPostgreSQL95Dialect.java

```diff
@@ -1,37 +0,0 @@
-package io.quarkus.hibernate.orm.runtime.dialect;
-
-import java.sql.DatabaseMetaData;
-import java.sql.SQLException;
-
-import org.hibernate.dialect.DatabaseVersion;
-import org.hibernate.dialect.PostgreSQL95Dialect;
-import org.hibernate.dialect.PostgreSQLDialect;
-import org.hibernate.engine.jdbc.env.spi.IdentifierCaseStrategy;
-import org.hibernate.engine.jdbc.env.spi.IdentifierHelper;
-import org.hibernate.engine.jdbc.env.spi.IdentifierHelperBuilder;
-
-/**
- * Subclass of PostgreSQL95Dialect fixing schema updates by considering unquoted identifiers as lower case.
- * This is PostgreSQL's behavior.
- * See https://github.com/quarkusio/quarkus/issues/1886
- *
- * @author Stephane Epardaud
- * @author Emmanuel Bernard emmanuel@hibernate.org
- */
-public class QuarkusPostgreSQL95Dialect extends PostgreSQLDialect {
-
-    public QuarkusPostgreSQL95Dialect() {
-        super(DatabaseVersion.make(9, 5));
-    }
-
-    @Override
-    public IdentifierHelper buildIdentifierHelper(IdentifierHelperBuilder builder, DatabaseMetaData dbMetaData)
-            throws SQLException {
-        // PostgreSQL considers unquoted identifiers lowercase
-        // https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS
-        builder.setUnquotedCaseStrategy(IdentifierCaseStrategy.LOWER);
```

Comment:
I confirm this should no longer be necessary: https://github.com/hibernate/hibernate-orm/blob/b2aa3c91f8edaac0db0f6e2fd527414b9d0e5eaf/hibernate-core/src/main/java/org/hibernate/dialect/PostgreSQLDialect.java#L857-L866